### PR TITLE
Oasis proxy

### PR DESCRIPTION
**Pull request by ethanbennett**
Integrates the Oasis Direct proxy with the testchain deployment script

Dai.js PR: https://github.com/makerdao/dai.js/pull/53

### CODE REVIEW

**Review by d-xo — APPROVED** [review]

File: out/DSAuth.abi [hunk 1]
```diff
@@ -0,0 +1 @@
+[{"constant":false,"inputs":[{"name":"owner_","type":"address"}],"name":"setOwner","outputs":[],"payable":false,"stateMutability":"nonpayable","type":"function"},{"constant":false,"inputs":[{"name":"authority_","type":"address"}],"name":"setAuthority","outputs":[],"payable":false,"stateMutability":"nonpayable","type":"function"},{"constant":true,"inputs":[],"name":"owner","outputs":[{"name":"","type":"address"}],"payable":false,"stateMutability":"view","type":"function"},{"constant":true,"inputs":[],"name":"authority","outputs":[{"name":"","type":"address"}],"payable":false,"stateMutability":"view","type":"function"},{"inputs":[],"payable":false,"stateMutability":"nonpayable","type":"constructor"},{"anonymous":false,"inputs":[{"indexed":true,"name":"authority","type":"address"}],"name":"LogSetAuthority","type":"event"},{"anonymous":false,"inputs":[{"indexed":true,"name":"owner","type":"address"}],"name":"LogSetOwner","type":"event"}]
```
Comment (levity) [inline out/DSAuth.abi]: this shouldn't be here

**Review by levity — APPROVED** [review]
LGTM once you remove the stray ABI

**Comment by levity** [comment]
But please don't merge until the corresponding dai.js branch is passing tests.

**Review by konstantinzolotarev — APPROVED** [review]
LGTM

**Pull request merged**